CMakeLists.txt

```diff
@@ -1,27 +1,42 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(octomap_msgs)
 
-find_package(catkin REQUIRED COMPONENTS message_generation geometry_msgs std_msgs)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-include_directories(include)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic")
+endif()
 
-add_message_files(
-  DIRECTORY msg
-  FILES Octomap.msg OctomapWithPose.msg
-  )
+find_package(ament_cmake REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(std_msgs REQUIRED)
+find_package(geometry_msgs REQUIRED)
 
-add_service_files(
-  DIRECTORY srv
-  FILES GetOctomap.srv BoundingBoxQuery.srv
-)
+# ament_export_include_directories(include)
 
-generate_messages(DEPENDENCIES std_msgs geometry_msgs)
+set(msg_files
+	"msg/Octomap.msg"
+	"msg/OctomapWithPose.msg"
+)
+set(srv_files
+	"srv/BoundingBoxQuery.srv"
+	"srv/GetOctomap.srv"
+)
 
-catkin_package(DEPENDS message_runtime geometry_msgs std_msgs
-               INCLUDE_DIRS include
+rosidl_generate_interfaces(${PROJECT_NAME}
+  ${msg_files}
+  ${srv_files}
+	DEPENDENCIES
+		std_msgs
+		geometry_msgs
 )
 
-install(DIRECTORY include/
-        DESTINATION include
-        FILES_MATCHING PATTERN "*.h"
-        PATTERN ".svn" EXCLUDE)
+ament_package()
+
+# install(DIRECTORY include/
```

Comment:
Do not commented out, if it's not useful remove it please

Reply:
Fixed on https://github.com/AcutronicRobotics/octomap_msgs/pull/2 https://github.com/AcutronicRobotics/octomap_msgs/pull/2/commits/d44688224b3deb4b0fe58851d36d213075a9fddc